src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -237,6 +212,45 @@ void SetHeaderGestures(in IView header)
 		headerView.GestureRecognizers.Add(tapGestureRecognizer);
 	}
 
+	static async void ForceUpdateLayoutSizeForItemsView(Expander expander)
+	{
+		if (expander.Header is null)
+		{
+			return;
+		}
+
+		Element element = expander;
+		var size = expander.IsExpanded
+				? expander.Measure(double.PositiveInfinity, double.PositiveInfinity, MeasureFlags.IncludeMargins).Request
+				: expander.Header.Measure(double.PositiveInfinity, double.PositiveInfinity);
+		while (element is not null)
+		{
+			if (element.Parent is ListView listView)
+			{
+				(element as Cell)?.ForceUpdateSize();
+			}
+#if IOS || MACCATALYST
+			else if (element is CollectionView collectionView)
+			{
+				var handler = collectionView.Handler as Microsoft.Maui.Controls.Handlers.Items.CollectionViewHandler;
+				var controller = handler?.GetType().BaseType?.BaseType?.BaseType?.BaseType?.BaseType?.GetProperty("Controller", BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.GetProperty);
+				var uiCollectionViewController = controller?.GetValue(handler) as UIKit.UICollectionViewController;
+				if (uiCollectionViewController?.CollectionView.CollectionViewLayout is UIKit.UICollectionViewFlowLayout layout)
+				{
+					layout.EstimatedItemSize = new CoreGraphics.CGSize(size.Width, size.Height);
+					layout.ItemSize = layout.EstimatedItemSize;
+					await Task.Delay(500);
+					layout.InvalidateLayout();
+				}
+
+				collectionView.InvalidateMeasureInternal(Microsoft.Maui.Controls.Internals.InvalidationTrigger.MeasureChanged);
+			}
+#endif
```

Comment:
Isn't there a code path needed for Android Windows
#elif ANDROID ....
#elif WINDOWS ....

Reply:
other platforms seems to work correct without this additional recalculation